src/audio/selector/selector.c

```diff
@@ -692,9 +699,84 @@ static void set_selector_params(struct comp_dev *dev,
 	buffer_release(source);
 }
 
+static int selector_verify_params(struct processing_module *mod,
+				  struct sof_ipc_stream_params *params)
+{
+	struct comp_dev *dev = mod->dev;
+	struct comp_data *cd = module_get_private_data(mod);
+	struct comp_buffer *buffer, *sinkb;
+	struct comp_buffer __sparse_cache *buffer_c, *sink_c;
+	uint32_t in_channels;
+	uint32_t out_channels;
+
+	comp_dbg(dev, "selector_verify_params()");
+
+	sinkb = list_first_item(&dev->bsink_list, struct comp_buffer,
+				source_list);
```

Comment:
I'd move this to around line 757 below, it isn't needed before that

Reply:
Done.